seeds/seeder.postgres.js

```diff
@@ -25,85 +32,94 @@ if (cluster.isMaster) {
   cluster.on('exit', () => {});
 }
 
-let printFinish = (client, url, dbName, startTime) => {
-  let endTime = new Date();
-  console.log('+ -----------------------');
-  console.log('| Completed seed.');
-  console.log(`| Inserted ${seedCount} items to ${url}/${dbName}`);
-  console.log(`| Start time: ${startTime}`);
-  console.log(`| End time: ${endTime}`);
-  console.log(`| Elapsed time: ${dateMath.diff(startTime, endTime, 'seconds', true)} seconds`);
-  console.log('\\* ----------------\\rm/--');
-
-  client.close();
-  process.exit();
-};
+let startTime = new Date();
+let bars = [];
+
+if (cluster.isMaster) {
+  printStart(url, dbName, startTime, seedCount);
+  bars = makeBars(numCPUs, seedCount, printEvery);
+
+  workers.forEach((worker) => {
+    worker.on('message', (msg) => {
+      let [barId, inserted, ips, finished] = msg.split(',');
+      bars[barId].tick({ inserted, ips });
+
+      if (JSON.parse(finished)) {
+        finishedProcesses += 1;
+        if (finishedProcesses === numCPUs) {
+          printFinish(url, dbName, startTime);
+          process.exit();
+        }
+      }
+    });
+  });
+}
 
-// Connection URL
-const url = 'mongodb://localhost:27017';
+let tick = (cluster.isMaster) ? (inserted, ips) => {
+  bars[0].tick({
+    inserted,
+    ips
+  });
+} : (inserted, insertsPerSec, finished) => {
+  process.send(`${process.env.barId},${inserted},${insertsPerSec},${finished}`);
+};
 
-// Database Name
-const dbName = 'descriptions';
+const db = pgp(`${url}/${dbName}`);
+
+let fields = [
+  ['id', 'INTEGER'],
+  ['author', 'TEXT'], // user is a reserved word in postgres
+  // ['shipdetails', 'TEXT']
```

Comment:
do you need this still?

Reply:
This was for a different schema that I was switching between. Since I've decided on Mongo, I don't need the Postgres seeder at all anymore